FLT/DedekindDomain/AdicValuation.lean

```diff
@@ -14,13 +14,16 @@ topological `A`-algebras. This file makes some progress towards this.
 
 ## Main theorems
 
-* `FiniteAdeleRing.closureAlgebraMapIntegers_eq_integers` : The closure of `A` in `K_v` is `𝒪_v`.
+* `HeightOneSpectrum.closureAlgebraMapIntegers_eq_integers` : The closure of `A` in `K_v` is `𝒪_v`.
```

Comment:
```suggestion
* `IsDedekindDomain.HeightOneSpectrum.closureAlgebraMapIntegers_eq_integers` : The closure of `A` in `K_v` is `𝒪_v`.
```
and the same for the other two, except that this might trigger the long line linter. I guess you could instead say "In the `IsDedekindDomain.HeightOneSpectrum` namespace:" and then drop HeightOneSpectrum